File: P2_Paga.c
// Programa 1
// Este programa calcula la el sueldo neto

#include <stdio.h>

int main() {
    printf("Hola, bienvenido a tu programa calculo de sueldo neto\n");

    printf("Introduzca las horas: ");
    float horas;
    scanf("%f", &horas);

    printf("Introduzca la tarifa horaria: ");
    float tarifa;
    scanf("%f", &tarifa);

    float iva = 0.16;

    float pago = horas * tarifa;

    float impuesto = pago * iva;

    float pagoNeto = pago - impuesto;

    printf("%f", &pagoNeto);

    return 0;
}

File: P2_Paga.m
% Programa 2
% Este programa calcula la el sueldo neto

clear all; % Borrar todas las variables creadas aun abiertas

clc; % Limpiar consola

disp("Hola, bienvenido a tu programa calculo de sueldo neto"); % Mostrar mensaje en la consola

horas = input("Introduzca las horas: "); % Solicitar entrada al usuario

tarifa = input("Introduzca la tarifa horaria: "); % Solicitar entrada al usuario

iva = 0.16;

pago = horas * tarifa;

impuesto = pago * iva;

pagoNeto = pago - impuesto;
